shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -311,11 +311,11 @@
   views_to_recomposite_.insert(view_id);
 }
 
-NSObject<FlutterPlatformView>* FlutterPlatformViewsController::GetPlatformViewByID(int view_id) {
+UIView* FlutterPlatformViewsController::GetPlatformViewByID(int view_id) {
   if (views_.empty()) {
     return nil;
   }
-  return views_[view_id].get();
+  return touch_interceptors_[view_id].get().subviews[0];
```

Comment:
This is dangerous. The view tree could change in the future and subviews[0] might not be the platform view.

Reply:
@cyanglaz I reverted to my original commit.